packages/components/src/color-palette/style.native.scss

```diff
@@ -42,3 +38,12 @@
 	letter-spacing: 1.25;
 	font-weight: 500;
 }
+
+.headerText {
+	color: $gray;
```

Comment:
The same color is used for both light and dark mode in header text. As far as I checked, it looks ok but wanted to highlight it in case we want to provide specific colors for each mode.

Reply:
Thanks for double-checking, I reused it the same way we do it for the block settings, the same color works for both light/dark mode.